scipy/special/_support_alternative_backends.py

```diff
@@ -53,25 +55,24 @@ def __f(*args, _f=_f, _xp=xp, **kwargs):
     return __f
 
 
-def _get_shape_dtype(*args, xp):
```

Comment:
This was going to happen in gh-22695, of course. Would you take a look at that again now that your comments have been addressed?